pylint/extensions/magic_number.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Checks for magic numbers instead of literals."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker, utils
+from pylint.checkers.base.comparison_checker import ComparisonChecker
+from pylint.interfaces import HIGH
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+LEGAL_MAGIC_NUMBERS = {-1, 0, 1, "", "__main__"}
```

Comment:
This should be a configurable option imo.

Reply:
good idea, do you mean from the pylintrc file? how do I do that?

Reply:
Yes or the CLI :) There's an example here : https://github.com/PyCQA/pylint/blob/main/pylint/checkers/classes/class_checker.py#L773